packages/frontend/src/config/routing/Routing.tsx

```diff
@@ -3,13 +3,17 @@ import { Route, Switch } from "react-router-dom";
 import NAVIGATION from "./NavigationPath";
 import SingleMoviePage from "../../features/videos/page/SingleMoviePage";
 import Movies from "../../features/videos/page/Movies";
+import MovieListView from "../../features/videos/page/MovieListView"
+import Temp from "../../features/videos/page/Temp"
 
 const Routing: React.FC = () => {
   return (
     <Switch>
       <Route path={`${NAVIGATION.MOVIES}/:id`} exact component={SingleMoviePage} />
       <Route path={NAVIGATION.MOVIES} exact component={Movies} />
-      <Route component={Movies} />
+      <Route path={NAVIGATION.MOVIELIST} exact component={MovieListView} />
+      <Route component={Temp} />
```

Comment:
co to?

Reply:
wrzucony tymczasowy komponent żeby ogarnąc redirecta

Reply:
zostawiłem żeby każdy mógł sprawdzić jak działa

Reply:
ok